org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/dom/XMLDocument.java

```diff
@@ -107,6 +108,10 @@ public NoNamespaceSchemaLocation getNoNamespaceSchemaLocation() {
 		return noNamespaceSchemaLocation;
 	}
 
+	public String getSchemaInstancePrefix() {
+		return this.schemaInstancePrefix;

```

Comment:
Call initializeReferencedGrammarIfNeeded(); before returning schemaInstancePrefix to be sure to update schemaInstancePrefix